R/tostring.R

```diff
@@ -557,6 +557,12 @@ setMethod("toString", "MatrixPrintForm", function(x,
 
     # retrieving titles and footers
     allts <- all_titles(x)
+
+    # Fix for ref_fnotes with \n characters XXX this does not count in the pagination
+    if (any(grepl("\n", ref_fnotes))) {
+      ref_fnotes <- unlist(strsplit(ref_fnotes, "\n", fixed = TRUE))
+    }
+
```

Comment:
this is a safenet, being it in toString, it will not update the number of lines for pagination but neither break (I do not expect this to be used at all in general)